project/Build.scala

```diff
@@ -112,6 +129,7 @@ object Scrooge extends Build {
     sharedSettingsWithoutScalaVersion ++
     Seq(
       scalaVersion := "2.11.8",
+      crossScalaVersions := scalaVersion.value +: Seq("2.12.0-M4"),
```

Comment:
Can we keep this consistent with how we do this in other projects? I.e., how we do it in util [here](https://github.com/twitter/util/blob/develop/project/Build.scala#L30).